src/AppInstallerCommonCore/Public/AppInstallerErrors.h

```diff
@@ -80,6 +80,15 @@
 #define APPINSTALLER_CLI_ERROR_LICENSE_NOT_ACCEPTED             ((HRESULT)0x8a150041)
 #define APPINSTALLER_CLI_ERROR_PROMPT_INPUT_ERROR               ((HRESULT)0x8a150042)
 #define APPINSTALLER_CLI_ERROR_INVALID_MSIEXEC_ARGUMENT         ((HRESULT)0x8a150043)
+#define APPINSTALLER_CLI_ERROR_INSTALL_PACKAGE_IN_USE           ((HRESULT)0x8a150044)

```

Comment:
I wonder if we shouldn't jump to some higher range here to intentionally keep these separate from the rest and give us the ability to add more on the future.